src/util/dbg_chain_id.h

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Please, add a justification for this split in the commit message.

Reply:
Maybe do it the other way around and split to `sss_chain_id_tevent.c/h`? Description in commit message is certainly welcome. This would also make it more distinct in the file name itself.

Reply:
I re-did this commit following Pavel's suggestion instead, and added justification to the commit message.